src/metpy/plots/declarative.py

```diff
@@ -1504,5 +1514,10 @@ def _build(self):
             else:
                 self.handle.plot_parameter(location, data[ob_type][subset], color=color)
         if self.vector_field[0] is not None:
-            self.handle.plot_barb(data[self.vector_field[0]][subset],
-                                  data[self.vector_field[1]][subset])
+            if self.vector_field_length is not None:
```

Comment:
Might this be better as:
```python
kwargs = {'length': self.vector_field_length} if self.vector_field_length is not None else {}
self.handle.plot_barb(data[self.vector_field[0]][subset],
                      data[self.vector_field[1]][subset],
                      **kwargs)
```

Reply:
I was wondering if there was a nice kwargs way...done!